docs/ref/utils.txt

```diff
@@ -947,20 +947,34 @@ appropriate entities.
     Returns ``True`` if ``value`` is naive, ``False`` if it is aware. This
     function assumes that ``value`` is a :class:`~datetime.datetime`.
 
-.. function:: make_aware(value, timezone=None)
+.. function:: make_aware(value, timezone=None, is_dst=None)
 
     Returns an aware :class:`~datetime.datetime` that represents the same
     point in time as ``value`` in ``timezone``, ``value`` being a naive
     :class:`~datetime.datetime`. If ``timezone`` is set to ``None``, it
     defaults to the :ref:`current time zone <default-current-time-zone>`.
 
-    This function can raise an exception if ``value`` doesn't exist or is
-    ambiguous because of DST transitions.
+    When ``pytz`` is available the exception ``pytz.AmbiguousTimeError``
+    will be raised if you try to make ``value`` aware
+    during a DST transition where the same time occurs twice (when reverting
+    from DST). Setting ``is_dst`` to ``True`` or ``False`` will avoid the exception
+    by choosing if the time is pre-transition or post-transition respectively.
+    ``is_dst`` has no effect when ``pytz`` is not installed.
+
+    When ``pytz`` is available the exception ``pytz.NonExistentTimeError``
+    will be raised if you try to make ``value`` aware during a DST transition
+    such that the time never occurred (when entering into DST). You can catch
```

Comment:
This paragraph (in contrast with the above paragraph) seems to imply that the `is_dst` parameter is not useful for avoiding `NonExistentTimeError`. But it is: if you pass `is_dst=[True|False]` to `tz.localize` you will never get a `NonExistentTimeError`. The way it resolves isn't quite as easy to explain as with `AmbiguousTimeError` - effectively you either get the time an hour earlier than the nonexistent time, or an hour later, depending which value you pass for `is_dst`.